test_runner/src/main/kotlin/ftl/util/CreashReporter.kt

```diff
@@ -0,0 +1,60 @@
+package ftl.util
+
+import ftl.config.FtlConstants
+import io.sentry.Sentry
+import java.io.File
+import java.util.UUID
+
+private const val FLANK_API_KEY = "https://f626934e7811480db91c40f62f5035f0@o475862.ingest.sentry.io/5514444"
+private const val GSUTIL_FOLDER = ".gsutil"
+private const val ANALYTICS_FILE = "analytics-uuid"
+private const val DISABLED = "DISABLED"
+
+const val SESSION_ID = "session.id"
+const val OS_NAME = "os.name"
+const val FLANK_VERSION = " flank.version"
+const val FLANK_REVISION = "flank.revision"
+const val DEVICE_SYSTEM = "device.system"
+const val TEST_TYPE = "test.type"
+
+val captureError by lazy {
+    initCrashReporter(FtlConstants.useMock)
+    ::notify
+}
+
+internal fun initCrashReporter(
+    useMock: Boolean,
+    rootPath: String = System.getProperty("user.home")
+) = when {
+    useMock -> null
+    analyticsFileExistAndIsDisabled(rootPath) -> null
+    else -> initializeCrashReportWrapper()
+}
+
+private fun analyticsFileExistAndIsDisabled(rootPath: String) =
```

Comment:
Hm...this name is not fully self-explanatory (for me). The logic here is to check if a user had disabled google analytics, flank should respect those settings.
How about `isGoogleAnalyticsDisabled`? Maybe not exactly the same but something explains why we check that file

Reply:
agree